data/pc/latest/proto.yml

```diff
@@ -1,4 +1,4 @@
-!version: 1.20.3
+!version: 1.20.5
```

Comment:
that latest file has the nice benefit of having a readable diff for this kind of pr